ert3/evaluator/_evaluator.py

```diff
@@ -91,7 +78,7 @@ def isIndeterminate(self) -> bool:
         # whether or not we cannot determine progress. Always false for the time being.
         return False
 
-    def setIndeterminate(self, indeterminate: Union[bool, None]):
+    def setIndeterminate(self, indeterminate: Union[bool, None]) -> None:
```

Comment:
I think this could safely be `raise NotImplementedError`. IIRC the `BaseRunModel` itself calls this API during updates. We don't have that yet, so let's just make that clear in code.